frontend/src/utils/helpers/convertStatus.ts

```diff
@@ -0,0 +1,23 @@
+import { ClientCourse } from 'types/clientCourse';
+import { COURSE_STATUSES } from 'constants/statuses';
+
+export const convertStatus = (status?: ClientCourse['status']): string => {
+  if (status === COURSE_STATUSES.started) {
+    return '37';
+  }
+  if (status === COURSE_STATUSES.testing) {
+    return '80';
+  }
+  if (status === COURSE_STATUSES.completed) {
+    return '100';
+  }
+  if (
+    status === COURSE_STATUSES.failed ||
+    status === COURSE_STATUSES.pending ||
+    status === COURSE_STATUSES.approved
+  ) {
+    return '0';
+  }
+
+  return <string>status;
+};
```

Comment:
```suggestion
import { ClientCourse } from 'types/clientCourse';
import { COURSE_STATUSES } from 'constants/statuses';

export const convertStatusToProgress = (status: ClientCourse['status']): string => {
  const progressValues = {
    [COURSE_STATUSES.started]: '37',
    [COURSE_STATUSES.testing]: '80',
    [COURSE_STATUSES.completed]: '100',
  };

  return progressValues[status] ?? '0';
};
```

you can impove it like this, if you want